README.md

```diff
@@ -12,22 +11,23 @@ Seriously, [check it out](http://www.trello.com/).
 
 [Full API documentation](http://www.rubydoc.info/gems/ruby-trello).
 
-## Installation
-
-```
-# gem install ruby-trello
-```
-
 Full Disclosure: This library is mostly complete, if you do find anything missing or not functioning as you expect it
-to, please [let us know](https://trello.com/card/spot-a-bug-report-it/4f092b2ee23cb6fe6d1aaabd/17).
+to, please just create an issue.
```

Comment:
Could we add the link to Github Issue here please?

Reply:
Sure. I have added it in #278